src/access_nri_intake/__init__.py

```
@@ -8,4 +8,16 @@
 __version__ = _version.get_versions()["version"]
 
 CATALOG_LOCATION = "/g/data/xp65/public/apps/access-nri-intake-catalog/catalog.yaml"
```

There's a trick we use for managing config files which is common in Django/Web Development which might be relevant/useful here:

```python
import os

CATALOG_LOCATION = os.environ.get("CATALOG_LOCATION", "/g/data/xp65/public/apps/access-nri-intake-catalog/catalog.yaml")
```

Basically, it lets us override the default with an environment variable, if found, falling back to the hardcoded default. It strikes me that it might be useful here, although we'd probably want to prefix all the environment variable names as a namespacing strategy, eg.:
```python
CATALOG_LOCATION = os.environ.get(
    "ACCESS_NRI_INTAKE_CATALOG_LOCATION", "/g/data/xp65/public/apps/access-nri-intake-catalog/catalog.yaml"
)